### PR TITLE
LIT2444 3978 6688

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
One new record and updates for two others.
https://github.com/BetaMasaheft/Documentation/issues/2135

### CODE REVIEW

File: new/LIT6688TeachingMysteries.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The general record for the cases when
+                  the version of the work cannot be immediately defined,
+                either it is the more common <ref type="work" corresp="LIT2444Temher">Tǝmhǝrta ḫǝbuʾāt</ref> or
+                or less common <ref type="work" corresp="LIT2444Temher">ʾƎlmasṭoʾagya</ref>.</p>
```
Comment (eu-genia) [inline new/LIT6688TeachingMysteries.xml:41]: LIT3978Elmastoagya?

File: 2001-3000/LIT2444Temher.xml [hunk 2]
```diff
@@ -58,7 +58,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <p>Not to be confused with the text also known as <ref type="work" corresp="LIT3978Elmastoagya"/>
```
Comment (eu-genia) [inline 2001-3000/LIT2444Temher.xml:58]: You need something here to clearly disambiguate from the new general record.

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6688TeachingMysteries.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT6688TeachingMysteries.xml:41]: Yes, thanks.

File: new/LIT6688TeachingMysteries.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6688TeachingMysteries" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Teaching of Mysteries (general record)</title>
```
Comment (thea-m) [inline new/LIT6688TeachingMysteries.xml:8]: en

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/LIT6688TeachingMysteries.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT6688TeachingMysteries.xml:8]: ok

File: 2001-3000/LIT2444Temher.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 2001-3000/LIT2444Temher.xml:58]: ok

File: 3001-4000/LIT3978Elmastoagya.xml [hunk 4]
```diff
@@ -52,6 +52,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                Whilst <persName ref="PRS5684JesusCh">Jesus Christ</persName>, as the central celebrated ﬁgure,
                was described and praised without his name being mentioned directly,
                the initiated in the mysteries of the Christian faith knew who was intended.</p>
+               <p>Not to be confused with the text also known as <ref type="work" corresp="LIT3978Elmastoagya"/>.
+               For texts in the versions than cannot be immediately identified, see
+               <ref type="work" corresp="LIT6688TeachingMysteries"/>.
+               </p>
+               <p>Not to be confused with the text also known as <ref type="work" corresp="LIT2444Temher"/>.
```
Comment (eu-genia) [inline 3001-4000/LIT3978Elmastoagya.xml:59]: twice the same?

Comment (DenisNosnitsin1970) [reply 3001-4000/LIT3978Elmastoagya.xml:56]: corrected

**Review by eu-genia — APPROVED** [review]
I guess now mss descriptions pointing to the relevant records have to be checked if the references are correct.

**Comment by DenisNosnitsin1970** [comment]
Yes, I will be doing that, there will be not many changes.

**Pull request merged**